src/styles/blocks/product-list.scss

```diff
@@ -0,0 +1,28 @@
+.product-list {
+  padding-top: 50px;
+  padding-bottom: 73px;
+
+  display: flex;
+  justify-content: center;
```

Comment:
If I delete one card, the left card shouldn't change their position
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/32794096/201057101-3e2bf905-d4f9-49d8-afef-6f5670956324.png">